src/neo-asm/ASM_Language/linker/BuildedFunction.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Neo.ASML.Linker
+{
+
+    public class BuildedFunction
+    {
+        public int addr;
+        public string name;
+        public List<BuildedOpCode> codes;
+
+        public int getFinalLength()
```

Comment:
Shouldn't we follow C# standards? (methods and properties with upper camel case)